partials/socials.hbs

```diff
@@ -1,5 +1,5 @@
-<div class="socials">
+<div class="socials row">
   {{#each socials}}
-    <a href="{{this}}" target="_blank"><i class="fa fa-{{@key}}"></i></a>
+    <a class="col-xs-4 col-sm-2 col-md-2" href="{{this}}" target="_blank"><i class="fa fa-{{@key}}"></i></a>
```

Comment:
@LucaLanziani Socials aren't daily posts. I think we can fix layout when we should have to put new icons.
Now is a row of 6 icons on desktop, two row of 3 icons on mobile. A new icon can be change the layout into 1x4 + 1x3 on both for example.

Let me know if keep the original layout implementing custom grid.

/cc @gabrielem